feat : 감상평 리스트 아이템 컴포넌트 제작

cit1566 opened the pull request:
## PR 유형

- [x] 새로운 기능 추가
- [ ] 버그 수정
- [ ] CSS 등 사용자 UI 디자인 변경
- [ ] 코드에 영향을 주지 않는 변경사항(오타 수정, 탭 사이즈 변경, 변수명 변경)
- [ ] 코드 리팩토링
- [ ] 주석 추가 및 수정
- [ ] 문서 수정
- [ ] 테스트 추가, 테스트 리팩토링
- [ ] 빌드 부분 혹은 패키지 매니저 수정
- [ ] 파일 혹은 폴더명 수정
- [ ] 파일 혹은 폴더 삭제

## PR 체크리스트

<!-- PR이 다음 요구 사항을 충족하는지 확인하세요. -->

- [] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

### PR 상세
- 아이템 리스트 컴포넌트 제작
## 이슈

<!-- 이슈 키워드와 함께 #을 입력한 후 이슈 번호를 선택해주세요. -->
<!-- 에시 : resolves #1 -->

resolves #

cit1566 committed:
init : 패키지 업데이트 설정 파일 변경

cit1566 committed:
feat : main-review-item 추가

cit1566 committed:
ci : 패키지 업데이트

cit1566 committed:
feat : 게시물 리스트 아이템 UI 제작

cit1566 committed:
Merge branch 'develop' into feature/mainListItemDesign/kim

cit1566 committed:
ci : next 최신 버전 업데이트

sion0221 reviewed (approved):
확인했습니다

chajiiiii commented on src/components/main-review-item/main-review-item.tsx:
```diff
@@ -0,0 +1,67 @@
+      <Link href="/" className={styles.posterLink}>
+        <Image
+          src="/image/noImage.png"
+          alt="poster"
```
대체 텍스트는 한국어로 하면 좋을 거 같습니다.

chajiiiii commented on src/components/main-review-item/main-review-item.tsx:
```diff
@@ -0,0 +1,67 @@
+          <Link href="/" className={styles.userId}>
+            <Image
+              src="/next-js.svg"
+              alt="userProfile"
```
대체 텍스트는 한국어로 하면 좋을 거 같습니다.

chajiiiii commented on src/styles/theme.css:
```diff
@@ -8,6 +8,10 @@
   --color-bg-dark: #f2f2f2;
+  --color-text-gray: #667085;
+  --color-text-purple: #6941c6;
+  --color-bolder-4: #ebebeb;
```
오타 수정 바랍니다. bolder -> border

chajiiiii commented on src/components/main-review-item/main-review-item.tsx:
```diff
@@ -0,0 +1,67 @@
+
+export default function MainReviewItem() {
+  return (
+    <section className={styles.mainReviewItem}>
```
카드 컴포넌트는 section 대신 article 사용을 추천 드립니다.
https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/article

chajiiiii commented on src/components/main-review-item/main-review-item.tsx:
```diff
@@ -0,0 +1,67 @@
+
+          {/* 좋아요 수 */}
+          <span className={styles.likes}>
+            <ThumbsUp size={20} color="#1ED534" />
```
클래스로 스타일을 설정하지 않고 인라인 스타일로 설정하신 이유가 따로 있을까요?

cit1566 replied:
css로 해당 아이콘의 색상 값 연결 했습니다~!

chajiiiii reviewed (commented):
리뷰 코멘트 달았습니다. 확인 부탁드립니다.

cit1566 committed:
style : PR 수정
- 대체 택스트 한글로 변경
- theme.css의 --color-border-4 속성값 오타 수정
- section -> article로 변경
- ThumbsUp 색상 값 css로 추가

cit1566 committed:
fix : page.tsx 원상복구
